### PR TITLE
fixed iTunes

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1564 .

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit be9453cf2360096c51e4cf3cbb34fa85cd3ff28a: Broken link result: [!] Error(s) found by the Links validator: 7 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06d-Testing-Data-Storage.md [hunk 1]
```diff
@@ -670,11 +670,13 @@ func excludeFileFromBackup(filePath: URL) -> Result<Bool, ExcludeFileError> {
...
+In order to test the backup, you obviously need to create one first. The most common way to create a backup of an iOS device is by using iTunes, which is available for Windows, Linux and of course macOS (till macOS Mojave). When creating a backup via iTunes you can always only backup the whole device and not select just a single app. Make sure that the option "Encrypt local backup" in iTunes is not set, so that the backup is stored in cleartext on your hard drive.
 
-After the iOS device has been backed up through iTunes you need to retrieve the file path of the backup, which are different locations on each OS. The official Apple documentation will help you to [locate backups of your iPhone, iPad, and iPod touch](https://support.apple.com/en-us/HT204215 "Locate backups of your iPhone, iPad, and iPod touch").
+On macOS Catalina onwards, iTunes is not available anymore. Managing of an iOS device, including updates, backup and restore has been moved to the Finder app. The approach remains the same, as described above.
```
Comment (cpholguera) [inline Document/0x06d-Testing-Data-Storage.md]:
```suggestion
> iTunes is not available anymore from macOS Catalina onwards. Managing of an iOS device, including updates, backup and restore has been moved to the Finder app. The approach remains the same, as described above.
```

File: Document/0x06d-Testing-Data-Storage.md [hunk 2]
```diff
@@ -597,7 +597,7 @@ If the information is masked by, for example, asterisks or dots, the app isn't l
 
 #### Overview
 
-iOS includes auto-backup features that create copies of the data stored on the device. On iOS, backups can be made through iTunes or the cloud (via the iCloud backup feature). In both cases, the backup includes nearly all data stored on the device except highly sensitive data such as Apple Pay information and Touch ID settings.
+iOS includes auto-backup features that create copies of the data stored on the device. Backups of iOS can be made locally to the device it's connected to through iTunes (till macOS Catalina) or Finder (macOS Catalina onwards), or the cloud (via the iCloud backup feature). In both cases, the backup includes nearly all data stored on the iOS device except highly sensitive data such as Apple Pay information and Touch ID settings.
```
Comment (cpholguera) [inline Document/0x06d-Testing-Data-Storage.md]: More active.
```suggestion
iOS includes auto-backup features that create copies of the data stored on the device. You can make iOS backups from your host computer by using iTunes (till macOS Catalina) or Finder (from macOS Catalina onwards), or via the iCloud backup feature. In both cases, the backup includes nearly all data stored on the iOS device except highly sensitive data such as Apple Pay information and Touch ID settings.
```

File: Document/0x06b-Basic-Security-Testing.md [hunk 3]
```diff
@@ -116,13 +120,13 @@ Jailbreaking an iOS device is becoming more and more complicated because Apple k
...
+iOS upgrades are based on a challenge-response process (generating as a result the named SHSH blobs). The device will allow the OS installation only if the response to the challenge is signed by Apple. This is what researchers call a "signing window", and it is the reason you can't simply store the OTA firmware package you downloaded and load it onto the device whenever you want to. During minor iOS upgrades, two versions may both be signed by Apple (the latest one, and the previous iOS version). This is the only situation in which you can downgrade the iOS device. You can check the current signing window and download OTA firmware from the [IPSW Downloads website](https://ipsw.me "IPSW Downloads").
```
[the remark this answers is not in the document]
Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: SHSH blobs is fine according to a quick google search

**Comment by sushi2k** [comment]
Thanks for cleaning @cpholguera. All issues resolved, please approve

**Comment by OMTGreleaser** [comment]
Results for commit f08baa9bd1e2aeaa7e8d046aecbad16c07d0aa86: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md [hunk 4]
```diff
@@ -116,13 +120,13 @@ Jailbreaking an iOS device is becoming more and more complicated because Apple k
...
+iOS upgrades are based on a challenge-response process (generating the named SHSH blobs as a result). The device will allow the OS installation only if the response to the challenge is signed by Apple. This is what researchers call a "signing window", and it is the reason you can't simply store the OTA firmware package you downloaded and load it onto the device whenever you want to. During minor iOS upgrades, two versions may both be signed by Apple (the latest one, and the previous iOS version). This is the only situation in which you can downgrade the iOS device. You can check the current signing window and download OTA firmware from the [IPSW Downloads website](https://ipsw.me "IPSW Downloads").
```
Comment (cpholguera) [inline Document/0x06b-Basic-Security-Testing.md]:
```suggestion
iOS upgrades are based on a challenge-response process (generating the so-called SHSH blobs as a result). The device will allow the OS installation only if the response to the challenge is signed by Apple. This is what researchers call a "signing window", and it is the reason you can't simply store the OTA firmware package you downloaded and load it onto the device whenever you want to. During minor iOS upgrades, two versions may both be signed by Apple (the latest one, and the previous iOS version). This is the only situation in which you can downgrade the iOS device. You can check the current signing window and download OTA firmware from the [IPSW Downloads website](https://ipsw.me "IPSW Downloads").
```

**Pull request merged**